src/main/java/com/ieltswise/entity/UserLessonData.java

```diff
@@ -24,8 +24,8 @@ public class UserLessonData {
     private Long userId;
     @Column(name = "EMAIL", unique = true, nullable = false)
     private String email;
-    @Column(name = "NAME")
-    private String name;
+    @Column(name = "NAME", columnDefinition = "VARCHAR(255) DEFAULT 'User'", nullable = false)
```

Comment:
What are the benefits of adding a default value to a database column?

Reply:
If we insert data into the database manually, without using our program, then the default value for the NAME column will be "User", not null